src/main/webapp/js/map_loader.js

```diff
@@ -0,0 +1,24 @@
+let map;
+function createMap(){
+  <!--create map object and place in map element-->
+  map = new google.maps.Map(document.getElementById('map'), {
+    center: {lat: 37.422, lng: -122.084},
+    zoom: 16
+  });
+
+<!--add T-rex marker-->

```

Comment:
Just as an fyi, for comments in javascript we use // for line comments, and /* text here */ for block comments. The comment style here is supposed to be for html, bit surprised it works tbh

Reply:
oh interesting. It's just what the IDE defaulted to. fixed!